### PR TITLE
Circle ability revamp

### LINKED ISSUE #634: Circle recasts and powers are inconsistent and often incorrect.
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: base

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
Holy and Arcane Circles have a 10 minute recast instead of 5 like they're supposed to, and they do not have their power reduced when the job is subbed. Warding and Ancient Circles appear to be correct, as do all of the existing modifiers for duration increase (all +90s for all four circles), along with the base power for all four circles from main job (should be 15% from main job, 5% when subbed).

### PR DESCRIPTION
**Pull request by illzbee**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Updates circle abilities recast time and subjob power to match retail

Closes: #634

### CODE REVIEW

File: scripts/globals/abilities/ancient_circle.lua [hunk 1]
```diff
@@ -14,7 +16,14 @@ ability_object.onAbilityCheck = function(player, target, ability)
 end
 
 ability_object.onUseAbility = function(player, target, ability)
-    xi.job_utils.dragoon.useAncientCircle(player, target, ability)
```
Comment (WinterSolstice8) [inline scripts/globals/abilities/ancient_circle.lua]: `xi.job_utils.dragoon.useAncientCircle` already handled the power. there was no need to change it. This is also a regression as it doesn't handle job points. Please revert this file.

**Review by WinterSolstice8 — CHANGES_REQUESTED** [review]

**Comment by WinterSolstice8** [comment]
While you're editing these, could you add in the missing job point power checks?

see
`scripts\globals\job_utils\dragoon.lua` function `xi.job_utils.dragoon.useAncientCircle` for how that's done

**Comment by illzbee** [comment]
> While you're editing these, could you add in the missing job point power checks?
> 
> see `scripts\globals\job_utils\dragoon.lua` function `xi.job_utils.dragoon.useAncientCircle` for how that's done

Added these in thanks for the tip to get it done!

**Comment by WinterSolstice8** [comment]
Hey so we didn't want the history of removing and re-adding the Ancient Circle file, so I rebased your commits to not have that. I squashed them so there's less commits as well to be cleaner. Your content was fine!

Git is difficult to use so you can reach out to us in discussion if you want to learn more about how to do those things.

**Review by WinterSolstice8 — APPROVED** [review]

**Review by claywar — APPROVED** [review]

**Pull request merged**